src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -154,16 +158,23 @@ where
     G: Scope<Timestamp = Timestamp>,
     S: SourceReader,
 {
-    let ((batches, source_upper_summaries), source_reader_token) =
-        source_reader_operator::<G, S>(config.clone(), source_connection, connection_context);
+    let (resume_stream, resume_token) = super::resumption::resumption_operator(config.clone());
+    let resume_stream = resume_stream.broadcast();
```

Comment:
nit: with how timely (currently) works, there should be no need for a `broadcast()` here.

Reply:
I was thinking about this on my bikeride this morning, as we don't send any data, we dont need this, yes!